maven-settings-template.xml

```diff
@@ -5,17 +5,32 @@
  <profiles>
     <profile>
       <id>release</id>
+
       <activation>
         <activeByDefault>true</activeByDefault>
       </activation>
+
       <repositories>
         <repository>
-          <id>confluent</id>
+          <id>confluent-other</id>
           <name>Confluent</name>
           <url>PACKAGES_MAVEN_URL</url>
         </repository>
       </repositories>
+
       <pluginRepositories>
+
+        <pluginRepository>
```

Comment:
What's the purpose of duplicating this?

Reply:
it's not actually duplicated - one has `<layout>default</layout>` the other doesn't. i'm not exactly sure why we need this but I needed that locally to get some things to work so figured I may as well do it here as well.

Reply:
But doesn't having two repositories with the same ID mean one of them is being ignored? I thought that was the basis of the other fix (renaming the repo at PACKAGE_MAVEN_URL).